<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>idparameter</class>
 <widget class="QWidget" name="idparameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>845</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>845</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QCALabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:MonoName</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:idstop.PROC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Undulator Control Mode </string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:liIdGetMode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Debugging Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Undulator Table Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Harmonic</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 127);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:liIdGetHarmonic</string>
   </property>
  </widget>
  <widget class="QCAComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:Harmonic</string>
   </property>
   <item>
    <property name="text">
     <string>1st Harmonic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3rd Harmonic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5th Harmonic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7th Harmonic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Max Flux</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Auto</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Inter. Order</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IdAutoOrder</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Spline Interpolation</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>450</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IdGetSpline</string>
   </property>
  </widget>
  <widget class="QCAComboBox" name="comboBox_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>440</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IdSetSpline</string>
   </property>
   <item>
    <property name="text">
     <string>Off</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>On</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:strIdStatus</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IdConfirmed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fixed Values for Preset Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>690</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Configuration File</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>750</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>Move ID</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:idstart.PROC</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>750</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Display</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:wfGetEnergy.PROC</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>680</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:loIdConfig</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>680</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:loIdConfig</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>680</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:loIdConfig</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>790</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Energy</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_44">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>820</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:aiIdGetCurenergy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_45">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>790</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Gap</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_46">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>820</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:GetIdGap</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_47">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>790</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Shift</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_48">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>820</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:GetIdShift</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_49">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>790</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Range(eV)</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_50">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>820</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:aiIdMinenergy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_51">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>790</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_52">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>820</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 127);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:idStatus</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_53">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>820</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_54">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>820</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:aiIdMaxenergy</string>
   </property>
  </widget>
  <widget class="QCALed" name="LedV1" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IdConfirmed</string>
   </property>
  </widget>
  <widget class="QCAComboBox" name="ComboBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IdDebugLevel</string>
   </property>
   <item>
    <property name="text">
     <string>Off</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>On</string>
    </property>
   </item>
  </widget>
  <widget class="QCAComboBox" name="ComboBox_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IdMode</string>
   </property>
   <item>
    <property name="text">
     <string>ID Off</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Preset Gap/Shift</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Preset Gap</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Preset Shift</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gap only</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Shift only</string>
    </property>
   </item>
  </widget>
  <widget class="QCAComboBox" name="ComboBox_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>410</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IdSetAutoOrder</string>
   </property>
   <item>
    <property name="text">
     <string>On</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Off</string>
    </property>
   </item>
  </widget>
  <widget class="QCAAxis" name="posAxis" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>360</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>in Position Band</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:IdInPosBand</string>
   </property>
   <property name="units" stdset="0">
    <string>mm</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IdInPosBand</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="forceAxis" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>360</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Dwell Timeout</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:IdDwellTimeout</string>
   </property>
   <property name="units" stdset="0">
    <string>mm</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IdDwellTimeout</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="energyAxis" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>360</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Energy</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:aiIdGetEnergy</string>
   </property>
   <property name="units" stdset="0">
    <string/>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:aiIdSetEnergy</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="slopeAxis" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>360</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Slope</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:aiIdSlope</string>
   </property>
   <property name="units" stdset="0">
    <string>mm/eV</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetaiIdSlope</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="offAxis" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>360</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Offset</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:aiIdOffset</string>
   </property>
   <property name="units" stdset="0">
    <string>eV</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetaiIdOffset</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="gapiAxis" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>360</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Gap intern. Order</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:liIdGetGapOrder</string>
   </property>
   <property name="units" stdset="0">
    <string>eV</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:liIdSetGapOrder</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="shiAxis" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>360</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Shift intern. Order</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:liIdGetShiftOrder</string>
   </property>
   <property name="units" stdset="0">
    <string>eV</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:liIdSetShiftOrder</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="fixgapAxis" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>360</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Fixed Gap</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:aiIdGetFixedGap</string>
   </property>
   <property name="units" stdset="0">
    <string/>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:aiIdSetFixedGap</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="fixshiftAxis" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>360</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Fixed Shift</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:aiIdGetFixedShift</string>
   </property>
   <property name="units" stdset="0">
    <string/>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:aiIdSetFixedShift</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="userAxis" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>710</y>
     <width>360</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>User Table Comment</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:siIdComment</string>
   </property>
   <property name="units" stdset="0">
    <string/>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetsiIdComment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Force Confirm Errors</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IdConfirmError</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="fixgapshiftAxis" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>360</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Fixed Gap/shift of Energy</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:aiIdGetFixedGap</string>
   </property>
   <property name="units" stdset="0">
    <string>eV(-)</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:aiIdGetFixedShift</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCALabel</class>
   <extends>QLabel</extends>
   <header>qcalabel.h</header>
  </customwidget>
  <customwidget>
   <class>QCAPushButton</class>
   <extends>QPushButton</extends>
   <header>qcapushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QCAAxis</class>
   <extends>QWidget</extends>
   <header>qcaaxis.h</header>
  </customwidget>
  <customwidget>
   <class>QCAComboBox</class>
   <extends>QComboBox</extends>
   <header>qcacombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QCALed</class>
   <extends>QWidget</extends>
   <header>qcaled.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
